pom.xml

```diff
@@ -1690,21 +1689,6 @@
         <version>${bouncycastle.version}</version>
         <scope>test</scope>
       </dependency>
-      <dependency>
```

Comment:
nit: Maybe add a comment to hadoop-minicluster somewhere to explain why we don't add direct kerby dependencies.